bin/ci-tests.sh

```diff
@@ -2,46 +2,35 @@
 
 TEST_TIMEOUT=15m
 
-# print a command and execute it
-show() {
-	echo "$@" >&2
-	eval "$@"
-}
-
-fatal() {
-	echo "$@" >&2
-	exit 1
-}
-
 PKGS="$(go list ./...)"
 
+# Support passing custom flags (-json, etc.)
+OPTS="$@"
+if [[ -z "$OPTS" ]]; then
+	OPTS="-race -count=1 -failfast -v"
+fi
+
 export PKG_PATH=${GOPATH}/src/github.com/TykTechnologies/tyk
 
+# exit on non-zero exit from go test/vet
+set -e
+
 # build Go-plugin used in tests
-go build -o ./test/goplugins/goplugins.so -buildmode=plugin ./test/goplugins || fatal "building Go-plugin failed"
+echo "Building go plugin"
+go build -o ./test/goplugins/goplugins.so -buildmode=plugin ./test/goplugins
 
 for pkg in ${PKGS}; do
     tags=""
-
-    # TODO: Remove skipRace variable after solving race conditions in tests.
-    skipRace=false
-    if [[ ${pkg} == *"grpc" ]]; then
-        skipRace=true
-    elif [[ ${pkg} == *"goplugin" ]]; then
-        skipRace=true
+    if [[ ${pkg} == *"goplugin" ]]; then
         tags="-tags 'goplugin'"
     fi
 
-    race="-race"
-
-    if [[ ${skipRace} = true ]]; then
-        race=""
-    fi
     coveragefile=`echo "$pkg" | awk -F/ '{print $NF}'`
```

Comment:
```
    coveragefile=`echo "$pkg" | awk -F/ '{print $NF}'`
                 ^-- SC2006 (style): Use $(...) notation instead of legacy backticks `...`.
```